test/test_torch.py

```diff
@@ -3874,10 +3874,21 @@ def test_diagflat(self):
         self._test_diagflat(self, dtype=torch.float32, device='cpu')
 
     def test_eye(self):
-        res1 = torch.eye(100, 100)
-        res2 = torch.Tensor()
-        torch.eye(100, 100, out=res2)
-        self.assertEqual(res1, res2)
+        for dtype, device in product(torch.testing.get_all_dtypes(), torch.testing.get_all_device_types()):
+            if dtype == torch.bfloat16:
+                continue
+
+            for n, m in product([3, 5, 7], repeat=2):                
```

Comment:
Trailing whitespace

Reply:
Thank you for pointing this out. I'll push a commit to rectify this.

Reply:
Thanks, @vishwakftw. I see we ignore it in the `.flake8` configuration.

Let me check if we can enable it easily...

Reply:
https://github.com/pytorch/pytorch/pull/19828: this was the PR that removed W291 from .flake8.

Reply:
Thanks! I added #24238 to re-enable it.